src/components/svg-canvas-graph.tsx

```diff
@@ -151,7 +151,7 @@ const SvgCanvas = () => {
                 }
             } else {
                 // the node is just placed and should not trigger any save, only display the details
-                dispatch(addSelected(node));
+                dispatch(setSelected(new Set([node])));
```

Comment:
I have added it when it is created.

Reply:
Oh, I see, so here should be a no-op? Is only changing this line to `dispatch(setSelected(new Set([node])));` works the same as your previous change?

Reply:
No-op. 

It is different between this and the previous one. Nodes may not have generated when the pointer is up.

Reply:
That makes sense. Let's remove this line and update the comment. Since there has been no feedback until now, I believe you can merge this whenever you want :)

Reply:
Done.